modules/bs.hitbox/data/bs.hitbox/tags/block/can_pass_through.json

```diff
@@ -1,4 +1,19 @@
 {
+  "__bookshelf__": {
+    "feature": false,
```

Comment:
To set to "yes", since we consider as a feature all the documented callable resources.